build_and_test.sh

```diff
@@ -19,17 +19,25 @@ source activate.sh
 
 echo "Building solution"
 
-dotnet build
+dotnet build -c Release
 
 echo "Testing solution"
 
 test_projects=( $( ls test/**/*Tests.csproj ) )
 
 for test_project in "${test_projects[@]}"
 do
-    # Capturing test diagnostic logs because of hanging build
-    # https://github.com/grpc/grpc-dotnet/pull/363
-    dotnet test $test_project --no-build --diag:artifacts/${test_project##*/}.log.txt
+    # "dotnet test" is hanging when it writes to console for an unknown reason
```

Comment:
Hmm weird but okay.

Reply:
do we believe this because of a bug in the preview version of .NET core or because of a bug in grpc-dotnet?

Reply:
I believe it is an issue in .NET Core test. Issue here - https://github.com/microsoft/vstest/issues/2080

I've added a link to it in the build script